code/src/UI/Controls/ImageEx.cs

```diff
@@ -70,7 +93,7 @@ private UIElement CreateFromXaml()
 
                 if (paths.Any())
                 {
-                    paths.ForEach(p => BindingOperations.SetBinding(p, Shapes.Path.FillProperty, CreateBinding(this, nameof(Foreground))));
+                    paths.ForEach(p => BindingOperations.SetBinding(p, Shapes.Shape.FillProperty, CreateBinding(this, nameof(Foreground))));
```

Comment:
I'm curious why this change? It doesn't change any behavior but just move a reference up the inheritance hierarchy. On its own, this line still looks fine, but in context, it's next to a block of code that works on "Shapes" and so the block of code about "Paths" now has a mention of "shapes" that could be potentially confusing and looks like a cut and paste error.